### ISSUE TITLE
DR: System-specific styling needs for our DH

### ISSUE THREAD

**Issue by changecourse** [mozilla-lockbox/lockbox-extension#662]
Alignment of our doorhanger to corresponding system defaults within the browser.

On macOS and Linux doorhangers have rounded corners. (2px corner radius)

On Windows doorhangers have sharp corners.

**Comment by jimporter** [comment]
As far as I can tell, Linux actually uses square corners everywhere. It's just Mac that's the weird one. 

Is there any implementation work to do here, or is this just a design task? The last I checked, we don't have any control over this, since it's something Firefox does.

**Comment by changecourse** [comment]
I was pulling from the spec... but if Linux doesn't do rounded corners for default firefox doorhangers, then I'll address with the Photon team...

https://design.firefox.com/photon/components/doorhangers.html 

@jimporter I put this in as an implementation task, but I guess that's an open question; is our app supposed to identify the current system and inherit any specific styles like this?

**Comment by jimporter** [comment]
@changecourse Yeah, let's file an issue for the Linux thing, since I don't see rounded corners on Linux and I don't think I ever have.

As for the doorhangers themselves, I don't think we have control over that. From the images people on macOS post, it seems like the corners are (used to be?) rounded. For example, [this one](https://github.com/mozilla-lockbox/lockbox-extension/pull/601#pullrequestreview-100880470) from 2 weeks ago. However, a [more-recent one](https://github.com/mozilla-lockbox/lockbox-extension/pull/619#pullrequestreview-103149475) looks bad (from 3 days ago).

Maybe there's a Firefox bug? As far as I know we haven't changed anything that would affect this between now and then.

All that said, there may be some merit in doing platform-specific styles on the account details dropdown in the management UI. That's supposed to look doorhanger-ish, right?

**Comment by changecourse** [comment]
Hmmmmm.... I did a little more digging, and installed SnoozeTabs via TestPilot... They now have squared off corners too on MacOS, and that was not the case previously (as far as I can recall)

I'll try and see if anyone else is experiencing this or if a Fx bug exists. 

> All that said, there may be some merit in doing platform-specific styles on the account details dropdown in the management UI. That's supposed to look doorhanger-ish, right?

Do you mean for the full-tab management UI? Are there areas in this view that feel too aligned to MacOS, is that what you're saying?

**Comment by jimporter** [comment]
> Do you mean for the full-tab management UI? Are there areas in this view that feel too aligned to MacOS, is that what you're saying?

Yes. I don't have a strong opinion on if it's "right" or not, but if we wanted that little dropdown to look like a "native" doorhanger, then this issue would probably be a good place to work on that.
